doc/changes/latest.rst

```diff
@@ -101,3 +101,5 @@ Changes
 - :bdg-dark:`Code` Reorder condition in internal call of :func:`nilearn.image.resample_img` to skip checking of array values if interpolation is ``nearest`` (:gh:`4571` by `Jason Kai`_).
 
 - :bdg-dark:`Code` Remove redundant sorting of ``np.unique(data)`` in internal call of :func:`nilearn.image.resample_img` when checking array values (:gh:`4571` by `Jason Kai`_).
+
+- :bdg-primary:`Doc` Add missing default values to the docstrings in 'nilearn/glm/first_level'(:gh:`4659` by `Anupriya Kumari`_).
```

Comment:
Let's keep it for now but I think that we should be able a single line in the changelog that mentions all the PRs you did.

Something like:

```rst
- :bdg-primary:`Doc` Add missing default values to docstrings (:gh:`4656`, :gh:`4659` by `Anupriya Kumari`_).
```